### DISCUSSION TITLE
BAN #1378

### DISCUSSION THREAD

**Discussion by RETRN1** [share121/inter-knot#1378] [其他]
![河蟹@1x](https://github.com/user-attachments/assets/9c0c0b2e-e4a8-4976-a186-051355f055fc)

**Comment by Yizakl** [comment]
是真的！

**Comment by luo-luo-o** [comment]
这个竟然是真的！！大家快来抢啊！！手慢了就没有啦！！

**Comment by lllliufeng** [comment]
还真是
<div align="right">——小尾巴，小尾巴，我是一条小尾巴</div>

**Comment by EhuanyingK** [comment]
这绳网怎么还带诈骗的，举办了

**Comment by xiaochenyu24** [comment]
我是没想到这里还有诈骗的

**Comment by su2569** [comment]
治安局提醒您：建议安装国家反诈中心
[![image](https://bkimg.cdn.bcebos.com/pic/6a63f6246b600c3387448165dc1f460fd9f9d72a0b58?x-bce-process=image/format,f_auto/resize,m_lfit,limit_1,h_1000)](https://bkimg.cdn.bcebos.com/pic/6a63f6246b600c3387448165dc1f460fd9f9d72a0b58?x-bce-process=image/format,f_auto/resize,m_lfit,limit_1,h_1000)
https://www.coolapk.com/apk/com.hicorenational.antifraud
https://apps.apple.com/cn/app/%E5%9B%BD%E5%AE%B6%E5%8F%8D%E8%AF%88%E4%B8%AD%E5%BF%83/id1552823102?l=en-GB
https://sj.qq.com/appdetail/com.hicorenational.antifraud?fromcase=40002&from_wxz=1

**Comment by VacuolePaoo** [comment]
关于 [EhuanyingK](https://github.com/EhuanyingK) 举报的帖子 “[福利]免费的10个加密母带快来领取吧！”  id: #1378 
中包含 [虚假、不准确或有意欺骗性信息](https://docs.github.com/zh/site-policy/acceptable-use-policies/github-misinformation-and-disinformation)，举报信息属实，绳网运营团队已经进行处理，让我们共同营造一个和谐的绳网氛围！

**Reply by Hongxuli2006** [reply]
真的是管理员啊

**Comment by lllliufeng** [comment]
狠活，点了
<div align="right">——小尾巴，小尾巴，我是一条小尾巴</div>

**Comment by storm09119** [comment]
可恶就差10抽保底！帖主和评论的托我要全部举报！

**Comment by lllliufeng** [comment]
被官方制裁力（悲
<div align="right">——小尾巴，小尾巴，我是一条小尾巴</div>

**Comment by EhuanyingK** [comment]
真的假的，不是你自己删的？

**Reply by lllliufeng** [reply]
帖子是edited by [VacuolePaoo](https://github.com/VacuolePaoo)，所以是管理员干了

**Comment by shenguojia123** [comment]
是真的，我领到了艾莲的地狱火

**Comment by SSTV-10068** [comment]
您好，您的委托可以挂在SSTV上的广告位上喔~
SSTV广告位价格实惠，广告醒目吸人，[欢迎大家抢购！](https://github.com/share121/inter-knot/discussions/1065)

**Comment by yigexie** [comment]
是真的！我拿到了
